src/components/Li.vue

```diff
@@ -0,0 +1,120 @@
+<template>
```

Comment:
this whole file is for a single element of the list, why don't we call it this way? would be easier to read.